src/kubeobjects/dynakube.go

```diff
@@ -10,19 +10,28 @@ import (
 )
 
 type DynakubeQuery struct {
-	clt       client.Client
-	namespace string
-	ctx       context.Context
+	clt        client.Client
+	kubeReader client.Reader
+	namespace  string
+	ctx        context.Context
```

Comment:
If I understand the `DynakubeQuery`'s purpose is to get information and not update it.

So having a `client` in it seems unnecessary or even scary.
It uses the `client` to `Get` objects, which we know is not the safest thing.

IMO we should just replace the `client` with the `apiReader` and not have both

Reply:
client removed